x/zkauth/keeper/keeper.go

```diff
@@ -1,41 +1,47 @@
 package keeper
 
 import (
-	"encoding/json"
 	"fmt"
-	"io"
-	"net/http"
-	"os"
-	"path/filepath"
 	"strconv"
 	"time"
 
+	"github.com/Finschia/ostracon/libs/log"
+	abci "github.com/tendermint/tendermint/abci/types"
+
 	"github.com/Finschia/finschia-sdk/baseapp"
 	"github.com/Finschia/finschia-sdk/codec"
 	storetypes "github.com/Finschia/finschia-sdk/store/types"
 	sdk "github.com/Finschia/finschia-sdk/types"
 	sdkerrors "github.com/Finschia/finschia-sdk/types/errors"
 	"github.com/Finschia/finschia-sdk/x/zkauth/types"
-	"github.com/Finschia/ostracon/libs/log"
-	abci "github.com/tendermint/tendermint/abci/types"
 )
 
 type Keeper struct {
-	cdc      codec.BinaryCodec
-	storeKey storetypes.StoreKey
-	router   *baseapp.MsgServiceRouter
+	cdc        codec.BinaryCodec
+	storeKey   storetypes.StoreKey
+	jwksMap    *types.JWKsMap       // JWK manager
+	zkVerifier types.ZKAuthVerifier // zkp verification key byte
+	homePath   string               // root directory of app config
```

Comment:
Would you like to delete it because it is not used?

Reply:
This path was supposed to be used to load the saved file of `jwksMap`, but now it is not used. Yes, it's good to remove in this case.